spec/graphql/mutations/subscriptions/create_with_override_spec.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::Subscriptions::CreateWithOverride, type: :graphql do
+  let(:membership) { create(:membership) }
+  let(:organization) { membership.organization }
+  let(:plan) { create(:plan, organization: organization) }
+  let(:customer) { create(:customer, organization: organization) }
+  let(:billable_metric) { create(:billable_metric, organization: organization) }
+  let(:mutation) do
```

Comment:
let blocks should be at the end of the let list, and if we can have some new line between blocks! 🙏